src/app/logout/page.tsx

```diff
@@ -0,0 +1,19 @@
+'use client'
+import { useKeyring } from "@w3ui/react-keyring"
+import { useRouter } from "next/navigation"
+import { useEffect } from "react"
+
+export default function LogoutPage () {
+  const [, { unloadAgent }] = useKeyring()
+  const router = useRouter()
+  useEffect(function () {
+    if (unloadAgent) {
+      async function logOutAndRedirect () {
+        await unloadAgent()
+        router.push('/')
+      }
+      logOutAndRedirect()
```

Comment:
you may want to `.catch()` errors and at least log them and/or support a prop that will get passed errors